meal-voucher-fr/build.gradle

```diff
@@ -12,14 +12,14 @@ plugins {
     id 'kotlin-parcelize'
 }
 
-ext.mavenArtifactId = "mealvoucher"
+ext.mavenArtifactId = "meal-voucher-fr"
 ext.mavenArtifactName = "Adyen checkout Meal Voucher component"
```

Comment:
We should add France to this line and the one under, right?

Reply:
Good one, added it.